examples/v2_boolean_org_enforce/variables.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "org_id" {
+  description = "Organization id for applying the policy at org level"
+  type        = string
+}
+
+# List of organization policies
+variable "org_policies" {
+  type        = map(any)
+  description = "Map of organization policies to be implemented at organization level"
+  default = {
```

Comment:
Let's hard code this in the example.

Reply:
`outputs` are defined based on these inputs. Isn't it ok to use `default` ?